src/app/docs/action-button/action-button-docs.component.html

```diff
@@ -7,6 +7,23 @@
     Action buttons create large buttons with icons, headers, and descriptions that present users with multiple, branching options to move forward with tasks
   </sky-docs-demo-page-summary>
 
+  <sky-docs-demo>
```

Comment:
@Blackbaud-ToddRoberts, should the demo include multiple action buttons? The usage guidance says `Don't use an action button by itself.`, so I'm thinking we probably want a few action buttons in the demo. Should we just include the three other action buttons from the current demo?

Reply:
@Blackbaud-AlexKingman, should the action button in the demo open a modal that says `Open action clicked` or something like that like the current demo does?

Reply:
Good point about the usage guidance. I'll just put the original demo back in the demo to make it quick.